File: Game/core/src/main/java/io/github/some_example_name/Question.java
package io.github.some_example_name;

public class Question {
    public String questionText;
    public String[] questionChoices;
    public int correctChoice;

    public Question(String questionText, String[] questionChoices, int correctChoice) {
        this.questionText = questionText;
        this.questionChoices = questionChoices;
        this.correctChoice = correctChoice;
    }
}

File: Reference/Code reference/Game Scripts/question.cs
using UnityEngine;

public class question
{
    public string questionText; // The question
    public string[] choices; // A list of 4 choices
    public int correctChoice; // The index of the correct choice (1-4)

    // Constructor
    public question(string questionText, string[] choices, int correctChoice)
    {
        this.questionText = questionText;
        this.choices = choices;
        this.correctChoice = correctChoice;
    }
}
